at_secondary/at_secondary_server/lib/src/notification/stats_notification_service.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:convert';
+
+import 'package:at_persistence_secondary_server/at_persistence_secondary_server.dart';
+import 'package:at_secondary/src/connection/inbound/inbound_connection_pool.dart';
+import 'package:at_secondary/src/server/at_secondary_config.dart';
+import 'package:at_secondary/src/server/at_secondary_impl.dart';
+import 'package:at_secondary/src/verb/handler/monitor_verb_handler.dart';
+import 'package:at_server_spec/at_server_spec.dart';
+import 'package:at_utils/at_logger.dart';
+
+/// [StatsNotificationService] is a singleton class that notifies the latest commitID
+/// to the active monitor connections.
+/// The schedule job runs at a time interval specified in [notification][_timeInterval]
+/// in [config.yaml]. Defaults to 15 seconds.
+/// The [schedule] method is invoked during the server start-up and should be called only
+/// once.
+class StatsNotificationService {
+  static final StatsNotificationService _singleton =
+      StatsNotificationService._internal();
+
+  StatsNotificationService._internal();
+
+  factory StatsNotificationService.getInstance() {
+    return _singleton;
+  }
+
+  final _logger = AtSignLogger('StatsNotificationService');
+  final _timeInterval = AtSecondaryConfig.statsNotificationTimeInterval;
```

Comment:
We can avoid a temp variable here and directly reference AtSecondaryConfig.statsNotificationTimeInterval